content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/deploy-corda-cluster.md

```diff
@@ -0,0 +1,388 @@
+```yaml
+imagePullSecrets:
+  - <IMAGE-PULL-SECRET-NAME>
+```
+
+### Replica Counts
+
+For high-availability, specify at least three replicas for each type of Corda worker:
+```yaml
+workers:
+  crypto:
+    replicaCount: 3
+  db:
+    replicaCount: 3
+  flow:
+    replicaCount: 3
+  membership:
+    replicaCount: 3
+  rpc:
+    replicaCount: 3
+  p2pGateway:
+    replicaCount: 3
+  p2pLinkManager:
+    replicaCount: 3
+```
+
+{{< note >}}
+Depending on your application workload, you may require additional replicas.
+{{< /note >}}
+
+### Resource Requests and Limits
+
+Specify a default set of resource requests and limits for the Corda containers:
+```yaml
+resources:
+  requests:
+    memory: 512Mi
+    cpu: 125m
+  limits:
+    memory: 2048Mi
+    cpu: 1000m
+```
+{{< note >}}
+It is particularly important to specify resource requests when using a Kubernetes cluster with auto-scaling to ensure that it scales appropriately when the Corda cluster is deployed.
+{{< /note >}}
+
+You can also override the default resource requests and limits separately for each type of Corda worker. For example, to increase the memory limit for flow workers:
+```yaml
+workers:
+  flow:
+    resource:
+      limit:
+        memory: 4096Mi
+```
+As with the number of replicas, you may need to adjust these values based on testing with your actual application workload.
+
+### REST API Load Balancer
+
+By default, the [REST API](../../developing/rest-api/rest-api.html) is exposed on an internal Kubernetes service. To enable access from outside the Kubernetes cluster, the API should be fronted by a load balancer. The Helm chart allows annotations to be specified to facilitate the creation of a load balancer by a cloud-platform specific controller. For example, the following configuration specifies that the [AWS Load Balancer Controller](https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html) fronts the REST API with a Network Load Balancer internal to the Virtual Private Cloud (VPC):
+```yaml
+workers:
+  rpc:
+    service:
+      type: LoadBalancer
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-internal: true
+        service.beta.kubernetes.io/aws-load-balancer-scheme: internal
+        service.beta.kubernetes.io/aws-load-balancer-type: nlb
+        external-dns.alpha.kubernetes.io/hostname: corda.example.com
+```
+
+### PostgreSQL
+
+The password for PostgreSQL can be specified directly as a Helm override but this is not recommended. Instead, create a Kubernetes secret containing the password with a key of password. By default, the install expects a database called `cordacluster` but this can be overridden. You can then define the PostgreSQL configuration as follows:
```

Comment:
```suggestion
The password for PostgreSQL can be specified directly as a Helm override but this is not recommended. Instead, create a Kubernetes secret containing the password with a key of *password*. By default, the install expects a database called `cordacluster` but this can be overridden. You can then define the PostgreSQL configuration as follows:
```

I italicised this as I presume it is meant to be the name of the key?